docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -25,13 +25,16 @@
 {{ end }}
 
 {{ $section := .CurrentSection }}
-{{ $api_section := .Site.GetPage "/docs/apis/_index.md" }}
+{{ $api_section := .Site.GetPage "/docs/apis/1.0/_index.md" }}
```

Comment:
We'll want to make this dynamic. Rather than having 2 hard-coded sections for the different versions, we want to check the `versions.json` file to see which ones are enabled, and load those.

Reply:
Updated